src/openrct2-ui/windows/Park.cpp

```diff
@@ -1613,7 +1613,7 @@ static void window_park_objective_paint(rct_window* w, rct_drawpixelinfo* dpi)
     // Objective outcome
     if (gScenarioCompletedCompanyValue != MONEY32_UNDEFINED)
     {
-        if ((uint32_t)gScenarioCompletedCompanyValue == 0x80000001)
+        if (static_cast<uint32_t>(gScenarioCompletedCompanyValue) == 0x80000001)
```

Comment:
I prefer a constant for this.
If you don't want to do that in this PR, could you create an issue?